Add more TS notification sounds

sinf opened the pull request:
Added some audio listed here: https://code.google.com/p/xcc/source/browse/trunk/xcc/data/ts%20mix%20description.txt

sinf committed:
Add more TS notification sounds

Mailaender commented:
:+1: Thanks!

reaperrr commented:
I appreciate the effort, but I don't think this should be merged in its current state.

1) Not a major problem, but there seem to be a few typos.

2) Most of the additions under sounds related to explosions, weapons and some other effects are only useful if you want to trigger a specific sound in a map without using a weapon/explosion, otherwise they're completely superflous because weapons, explosions etc. call the ```.aud``` files directly and don't use notifications.

3) Another potential problem is that this contains all Firestorm speeches and sounds. To my knowledge, we're concentrating on vanilla TS initially, and if our content download/installation doesn't take this into consideration, it might result in crashes if expand01.mix and its content are missing and some mapper tries to add a sound because we had it defined in notifications.yaml. Additionally, this bloats the yaml with sound and speech definitions we currently don't need.

4) This is simply too much for a single yaml, in my opinion. I'd at least split into speech and sounds, and then sub-split between ```generic and ```singleplayer```, since - as mentioned above - most of these sounds and speeches will only be used in map triggers (if at all), and I dare to say those won't play much of a role for quite some time to come.


To be perfectly honest, I'd prefer to add notifications only in moderation and when needed over this "slap everything in" approach. In my opinion, such a monster yaml would actually make things harder for us as long as the mod is still under development.

penev92 commented:
I agree with the first part of @reaperrr's comment. Explosions and other sound effects have no place in `notifications.yaml`, and our TS mod should not contain any FS assets.
That being said, I don't agree with the last part - not "registering" assets in mod YAMLs has lead to enough headaches in TD, RA and D2k. We should have everything available for when we need it, not make people have to go through the mix files searching for the 1 sound they need every time they need something new.
All TS voice notifications should be added here. Nothing more, nothing less. Splitting into separate YAML files might be good, so once we see how many all notifications are we can discuss that.

sinf commented:
I removed all speech notifications not present in either speech01.mix or speech02.mix (only vanilla TS speech remain) and also removed explosion/weapon/misc effects.

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		SpyLostMissionFailed: 01-n210
+		StartGame: 00-i200
+		StructureSold: 00-i228
+		SubterrainianUnitDetected: 00-i174
```
Typos, should be 'Subterranean'

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		FireStormDefenseReady: 00-i162
+		FourMinutesRemaining: 00-i128
+		HarvesterAttack: 00-i090
+		InsufficantFunds: 00-i022
```
Insufficient

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		CriticalStructureLost: 00-i196
+		CriticalUnitLost: 00-i194
+		EmPulseCannonReady: 00-i158
+		EstablishingBattleControllsStandBy: 01-n006
```
Controls with only one 'l'.

penev92 replied:
Shouldn't that be "Control" as well?

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		BuildingInfiltrated: 00-i014
+		Cancelled: 00-i220
+		ChemicalMissileReady: 00-i152
+		CivillianKilled: 00-n018
```
Civilian (one l too much).

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		AllianceFormed: 00-i304
+		BaseAttack: 00-i082
+		BaseDefensesOffline: 00-i180
+		BasePerimiterHasBeenBreached: 01-n005
```
Perimeter

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		PlayerWasDefeated: 00-i268
+		PrimaryBuildingSelected: 00-i034
+		PrimaryObjectiveAchieved: 00-i100
+		ProductionFacillityDestroyed: 01-n233
```
Facility

reaperrr commented on mods/ts/audio/speech-generic.yaml:
```diff
@@ -0,0 +1,81 @@
+		PrimaryBuildingSelected: 00-i034
+		PrimaryObjectiveAchieved: 00-i100
+		ProductionFacillityDestroyed: 01-n233
+		QuaterneryObjectiveAchieved: 00-i106
```
Quarternary

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+Speech:
+	Notifications:
+		AirPowerInnefectiveDuringIonStorms: 00-n072
```
Ineffective

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		CaptureNodTechCenterToRecoverTheCrystals: 00-n134
+		CaptureTheTvStationToTheEast: 01-n020
+		CapturingTheLocalRadarTowers: 01-n038
+		CivillianCityIsUnderAttack: 00-n255
```
Civilian

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		CaptureTheTvStationToTheEast: 01-n020
+		CapturingTheLocalRadarTowers: 01-n038
+		CivillianCityIsUnderAttack: 00-n255
+		CivilliansEvacuatedObjectiveComplete: 00-n024
```
Civilians

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		DontShootICanTellYouGdisEvacLocations: 47-n000
+		EnemySamSitesDetected: 00-n084
+		EtaForAirstrikeIs5Minutes: 35-n004
+		EyeOfTheStormHasBeenEnteredEqupimentAtMaximumEfficency: 00-n241
```
Efficiency

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		EnemySamSitesDetected: 00-n084
+		EtaForAirstrikeIs5Minutes: 35-n004
+		EyeOfTheStormHasBeenEnteredEqupimentAtMaximumEfficency: 00-n241
+		FighterProductionFacillityDestroyedMissionComplete: 00-n210
```
Facility

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		EtaForAirstrikeIs5Minutes: 35-n004
+		EyeOfTheStormHasBeenEnteredEqupimentAtMaximumEfficency: 00-n241
+		FighterProductionFacillityDestroyedMissionComplete: 00-n210
+		FighterProductionFacillityLocatedDropshipsEnRoute: 00-n208
```
Same here

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		HassanIsEscaping: 38-n010
+		HereComesKanesWelcomingComittee: 33-n022
+		HereTheyCome: 33-n008
+		IAmFreeNowItsPabackTimeHassan: 39-n000
```
Pa'y'back

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		IonStormFormingOverThisSectorPilotReturnToBase: 00-n140
+		IonStornWillDisableFighters: 00-n192
+		IsThereAnyTechLeftAtTheNodBase: 33-n010
+		ItSeamsThatCretainStructuresAttractLighting: 01-n054
```
Seems

penev92 replied:
Certain

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		IonStornWillDisableFighters: 00-n192
+		IsThereAnyTechLeftAtTheNodBase: 33-n010
+		ItSeamsThatCretainStructuresAttractLighting: 01-n054
+		ItWouldBeAdvantagesToDestroyBridges: 00-n042
```
Advantageous

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		KodiakUnderAttack: 00-n236
+		LaserTurretsRunForIt: 38-n006
+		LetsGetHimToTheConvoyPoint: 38-n020
+		LooksLikeTheyToreThisPlaceAppartAlready: 33-n004
```
Apart

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		McnealEscapedMissionFailed: 01-n234
+		McnealKilledMissionFailed: 01-n230
+		McvHasArrivedToTheSouthEast: 01-n040
+		MetiorStormApproaching: 00-i178
```
Meteor

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		MutantsLocated: 01-n128
+		MutantsLostMissionFailed: 00-n158
+		NegativeICanMakeIt: 35-n010
+		NodHasInstalledSeveralIcbmsToTakeOutThePhilladelphia: 00-n252
```
Philadelphia

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		OurWorkHereIsDoneWereOutOfHere: 10-n044
+		PerimeterDeactivated: 00-n112
+		PerimiterDeactivatedCommenceAttack: 01-n262
+		PhilladelphiaInRangeIcbmLaunchDetectedMissionFailed: 00-n248
```
Philadelphia

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		PrepairForDecimation: 01-i362
+		PreventGdisEvacuationAtAllCosts: 01-n102
+		PreventThatTrainFromLeaving: 01-n076
+		ProbobleObjectiveIsRescueOfMutantPrisoners: 01-n322
```
Probable

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		ReinforcementsArrived: 00-i038
+		ReinforcementsEnRoute: 00-n132
+		ReinforcementsInbound: 00-n114
+		ResearchFacillityDestroyed: 01-n134
```
Facility

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		ReinforcementsEnRoute: 00-n132
+		ReinforcementsInbound: 00-n114
+		ResearchFacillityDestroyed: 01-n134
+		ResearchFacillityLocated: 01-n132
```
Same here

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		RetreatIsYourOnlyLogicalOption: 00-i346
+		SamSitesDestroyedDropshipsInbound: 00-n088
+		SiteSecureObjectiveComplete: 00-n032
+		SoMuchForSutlty: 10-n038
```
Subtlety

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		SurrenderIsPreferableAgainstTheOddsYouAreFacing: 00-i372
+		SurrenderIsYourOnlyLogicalOption: 00-i348
+		TakePointSoldier: 38-n022
+		TascitusHasBeenAquired: 01-n066
```
Tacitus

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		ThanksForYourHelp: 37-n000
+		ThanksWellEnjoyThis: 36-n002
+		ThatTrainGoesToThePowerGridWeToldYouAboutGetUsOnBoard: 36-n000
+		TheCavelryHasArrived: 33-n014
```
Cavalry

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		ThatTrainGoesToThePowerGridWeToldYouAboutGetUsOnBoard: 36-n000
+		TheCavelryHasArrived: 33-n014
+		TheCreatureIsTheTiberiumSubstanceWeSeekUseIt: 01-n174
+		TheIceWontSupportTheWeightOfMultipleVechiclesCrossing: 38-n028
```
Vehicles

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TheCreatureIsTheTiberiumSubstanceWeSeekUseIt: 01-n174
+		TheIceWontSupportTheWeightOfMultipleVechiclesCrossing: 38-n028
+		TheIonCannonIsOurs: 01-n270
+		TheMedicalFacillityIsLocatedToTheEast: 10-n032
```
Facility

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TheIceWontSupportTheWeightOfMultipleVechiclesCrossing: 38-n028
+		TheIonCannonIsOurs: 01-n270
+		TheMedicalFacillityIsLocatedToTheEast: 10-n032
+		TheMedicalFacillityIsLocatedToTheWest: 10-n034
```
Same here

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TheMedicalFacillityIsLocatedToTheWest: 10-n034
+		TheMutantsHaveTurnedOnUs: 38-n016
+		TheSacrificeOfTheManyIsThePleasureOfTheFew: 01-i368
+		TheTraitorsAreCommingBlowTheBridge: 38-n008
```
Coming

reaperrr replied:
Coming

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TheyMayBeOnToUs: 37-n002
+		TheyveSeenUsFallBack: 10-n046
+		ThreeMinutesRemaining: 00-i130
+		TiberiamIsLethalToUnprotectedInfantry: 38-n002
```
Tiberium

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TheyveSeenUsFallBack: 10-n046
+		ThreeMinutesRemaining: 00-i130
+		TiberiamIsLethalToUnprotectedInfantry: 38-n002
+		TiberiumIsHazzardusToUnprotectedInfantry: 01-n009
```
Hazardous

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		TimeToEraseTheHumanFactorFromThisEquasion: 00-i360
+		ToBuildOrTrain: 01-n007
+		ToCaptureABuilding: 01-n026
+		ToDeployAVechile: 01-n027
```
Vehicle

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		WeHaveBeenBetrayedGdiKidnappedTratos: 36-n008
+		WeHaveBeenTouchedByTheSpiritHandOfKane: 40-n000
+		WeJustGotWordFromGeneralHassanHesMovingUpNorthToCairo: 41-n006
+		WeMustGetTratosOutOfThatFacillity: 10-n036
```
Facility

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		WeMustGetTratosOutOfThatFacillity: 10-n036
+		WeMustGetTratosToTheTransport: 10-n042
+		WeNeedMoreMenTrainThemFromTheHandOfNod: 38-n004
+		WeTireOfYourInsignificantDefience: 01-i366
```
Defiance

reaperrr commented on mods/ts/audio/speech-singleplayer.yaml:
```diff
@@ -0,0 +1,228 @@
+		YourDefeatIsAtHand2: 01-i352
+		YourDefeatIsAtHand: 00-i352
+		YourForcesHaveBeenDetected: 01-n100
+		YourProbobilityOfSucessIsDropping: 00-i370
```
Probability

reaperrr commented:
I think you could actually rename sounds-singleplayer.yaml to sounds-generic.yaml, as most of those sounds already are or probably will be used for UI/multiplayer as well.
